lms/templates/courseware/legacy_instructor_dashboard.html

```diff
@@ -329,8 +329,14 @@ <h2 class="navbar">[ <a href="#" onclick="goto('Grades');" class="${modeflag.get
     </div>
     % endif
 
-    <input type="submit" name="action" value="List enrolled students" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}">
-    <input type="submit" name="action" value="List students who may enroll but may not have yet signed up" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}" >
+    <p>
+      <input type="submit" name="action" value="List enrolled students" class="${'is-disabled' if disable_buttons else ''}" aria-disabled="${'true' if disable_buttons else 'false'}">
```

Comment:
Addressed in 71d2574.

Screenshot:

![legacy-dashboard-button-removed](https://cloud.githubusercontent.com/assets/961441/7910522/ce4e18b8-0855-11e5-847b-dc6e8af2f4be.png)